src/WinGetServer/WinGetServerManualActivation_Client.cpp

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "WinGetServer.h"
+#include "Utils.h"
+
+#include <wil/com.h>
+#include <wil/result.h>
+#include <wil/safecast.h>
+
+#include <memory>
+#include <mutex>
+#include <string>
+#include <shtypes.h>
+#include <filesystem>
+#include <shlobj_core.h>

```

Comment:
Nit: Since more than one files include these headers, we probably want to put them in pch.h
Also move ones from WinMain.cpp to pch.h too